<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Icon/apple.png</normaloff>:/Icon/apple.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Bài 1</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Icon/1 (52).png</normaloff>:/Icon/1 (52).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Bài 2</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Icon/1 (43).png</normaloff>:/Icon/1 (43).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Bài 3</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Icon/1 (32).png</normaloff>:/Icon/1 (32).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>681</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#fd052b;&quot;&gt;Bài 1: Sắp xếp kiểu chèn nhị phân&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#aaffff;&quot;&gt;Bài 2: Thuật toán Euclid&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ff7f;&quot;&gt;Bài 3:Sinh hoán vị liền sau của một hoán vị cho trước&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButton1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCh_n_b_i_t_pk">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionB_i_1"/>
    <addaction name="actionB_i_2"/>
    <addaction name="actionB_i_3"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionQT"/>
    <addaction name="actionUs"/>
   </widget>
   <addaction name="menuCh_n_b_i_t_pk"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionB_i_1">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</iconset>
   </property>
   <property name="text">
    <string>Bài 1</string>
   </property>
  </action>
  <action name="actionB_i_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</iconset>
   </property>
   <property name="text">
    <string>Bài 2</string>
   </property>
  </action>
  <action name="actionB_i_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</normaloff>:/Icon/1465032999_apple-corp-corporation-logo-logotype.ico</iconset>
   </property>
   <property name="text">
    <string>Bài 3</string>
   </property>
  </action>
  <action name="action_E">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/exit.png</normaloff>:/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQT">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/qt.png</normaloff>:/Icon/qt.png</iconset>
   </property>
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionUs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/1 (29).png</normaloff>:/Icon/1 (29).png</iconset>
   </property>
   <property name="text">
    <string>EvilTeam</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/exit.png</normaloff>:/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icon/exit.png</normaloff>:/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
